nextcord/ext/help_commands/__init__.py

```diff
@@ -1,10 +1,8 @@
+from .embedded import EmbeddedHelpCommand
 from .errors import MissingDependencyError
 from .paginated import PaginatedHelpCommand
 
 # Needed for the setup.py script
 __version__ = "0.0.1"
 
-__all__ = (
-    "MissingDependencyError",
-    "PaginatedHelpCommand",
-)
+__all__ = ("MissingDependencyError", "PaginatedHelpCommand", "EmbeddedHelpCommand")
```

Comment:
Preferably, this should be on 3 separate lines with commas after each.

Makes it easier to extend the list without changing as much.

Reply:
Black really want's to put these onto the same line for some reason.

```
Running black in all files................................................................Failed
- hook id: black
- files were modified by this hook
reformatted nextcord\ext\help_commands\__init__.py
```

Reply:
Make sure there's a comma at the end of each line